src/wallet/wallet.cpp

```diff
@@ -2160,6 +2160,9 @@ bool CWalletTx::SubmitMemoryPoolAndRelay(std::string& err_string, bool relay)
     if (!pwallet->GetBroadcastTransactions()) return false;
     // Don't relay abandoned transactions
     if (isAbandoned()) return false;
+    // Don't try to submit coinbase transactions. These would fail anyway but would
+    // cause log spam.
+    if (IsCoinBase()) return false;
```

Comment:
Sorry for that. You may need also to restore the `GetDepthInMainChain`, if you follow what @kallewoof was saying non-coinbase confirmed txn were also in log spams.

Reply:
No, I don't believe non-coinbase confirmed txn were in log spams but there were thousands so I can't easily verify.